### PR TITLE
DEPLOY: Move prod to use folder flows

### PR DESCRIPTION
**Pull request by gmatthews20**

### CODE REVIEW

**Review by anish-mudaraddi — APPROVED** [review]
Worked on this together and I'm happy with the changes - we're just mirroring staging here

File: clusters/prod/management/infra-values.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+openstack-cluster:
+  controlPlane:
+    machineCount: 3
```
Comment (DavidFair) [inline clusters/prod/management/infra-values.yaml:3]: This should be 5, actually we should drop this and use the chart default unless we need to tailor it

File: clusters/prod/management/infra-values.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  nodeGroups:
+    - name: default-md-0
+      machineCount: 2
+
+  nodeGroupDefaults:
+    machineFlavor: l3.nano
```
Comment (DavidFair) [inline clusters/prod/management/infra-values.yaml:10]: ^

File: clusters/prod/worker/infra-values.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+openstack-cluster:
+  controlPlane:
+    machineCount: 3
+
+  nodeGroups:
+    - name: default-md-0
+      machineCount: 2
+
+  nodeGroupDefaults:
+    machineFlavor: l3.nano
```
Comment (DavidFair) [inline clusters/prod/worker/infra-values.yaml:10]: ^

File: secrets/prod/management/.sops.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+creation_rules:
+  - unencrypted_regex: "^(apiVersion|metadata|kind|type)$"
+    key_groups:
+      - age:
+          # prod-rotate-key
+          - age1cq92796c46d06s43t079xc89exe4vd52rh30c9mcafmne62dxyhqrupl4l
+          # staging-argo-key
+          - age1mqqfadar4vz8yfsvfd540nkhwjvxuzmf3hgr95ftp7d0dqnm7ycsfue3kv
```
Comment (DavidFair) [inline secrets/prod/management/.sops.yaml:8]: Is this the staging key? It should be a dedicated prod key

**Review by DavidFair — CHANGES_REQUESTED** [review]

File: secrets/prod/management/.sops.yaml (hunk 4 above)
Comment (gmatthews20) [reply secrets/prod/management/.sops.yaml:8]: This is the prod key, I have updates the comment

**Review by anish-mudaraddi — APPROVED** [review]

**Review by DavidFair — APPROVED** [review]

**Pull request merged**